src/rviz/default_plugin/markers/marker_selection_handler.cpp

```diff
@@ -140,7 +136,7 @@ void MarkerSelectionHandler::createProperties( const Picked& obj, Property* pare
   orientation_property_ = new QuaternionProperty( "Orientation", getOrientation(), "", group );
   orientation_property_->setReadOnly( true );
 
-  scale_property_ = new MarkerScaleProperty( "Scale", getScale(), marker_->getMessage()->type, "", group );
```

Comment:
There is no need to duplicate code for a new `MarkerScaleProperty`, which actually behaves like a VectorProperty.
It's possible to rename (and hide) the fields.